c/polyjuice.h

```diff
@@ -1410,16 +1426,10 @@ int run_polyjuice() {
     return clean_evmc_result_and_return(&res, -1);
   }
   if (msg.gas < res.gas_left) {
```

Comment:
question: when will such a condition happen? (what does this condition mean)

Reply:
It was some debugging code.
will remove this unreachable if branch.